Packs/Akamai_SIEM/Integrations/Akamai_SIEM/Akamai_SIEM_test.py

```diff
@@ -276,6 +276,15 @@ def test_fetch_events_command_no_results(self, mocker, client, requests_mock):
         assert total_events_count == 0
         assert last_offset == "318d8"
 
+    def test_attach_offset_param_to_url(self, client):
+        """Test that the execute_get_events_request function doesn't encode the offset's ; into %3B."""
+        params = {"offset": "das2;test", "limit": 100}
+        try:
+            client.execute_get_events_request(params, 50170)
+        except Exception as e:
```

Comment:
whats this exception? is this actually trying to make a request? lets use request mock and make sure it doesnt actually use the network